app/connectors/MessageConnector.scala

```diff
@@ -32,7 +33,7 @@ class MessageConnector @Inject()(http: HttpClient, appConfig: AppConfig) extends
 
   def get(arrivalId: String, messageId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Either[HttpResponse, MovementMessage]] = {
     val url = rootUrl + s"/arrivals/$arrivalId/messages/$messageId"
-    http.GET[HttpResponse](url)(CustomHttpReader, implicitly, implicitly).map { r =>
+    http.GET[HttpResponse](url, queryParams = Seq(), headers = Seq(HeaderNames.CONTENT_TYPE -> "application/json"))(CustomHttpReader, implicitly, implicitly).map { r =>
```

Comment:
If we're going to do this in all connectors, it might be worth having a base connector that can define a def for addHeaders that can take in the content type as a MimeType (see the ArrivalConnector for the type). That should tidy up some of this and remove the magic strings

Reply:
For now can we add headers as a private val outside the methods and reference this in the methods?

Reply:
I'm confused, that's practically the same quantity of work.

Reply:
OK. So we could introduce a BaseConnector trait with a headers val?

Reply:
Added BaseConnector trait